nix/ocaml.nix

```diff
@@ -147,7 +147,7 @@ let
         MINA_COMMIT_DATE = "<unknown>";
         MINA_BRANCH = "<unknown>";
 
-        DUNE_PROFILE = "devnet";
+        DUNE_PROFILE = "dev";
```

Comment:
Good catch

Reply:
I think it might need to depend on the concrete type of building one's doing, but that's a task for the future.